src/main/java/codes/biscuit/skyblockaddons/features/tabtimers/TabEffectManager.java

```diff
@@ -21,6 +21,8 @@
  */
 public class TabEffectManager {
 
+    private static final Pattern SERVER_ID_REGEX = Pattern.compile("[0-9]{2}/[0-9]{2}/[0-9]{2}\\s\\s(mini[0-9]{1,3}[A-Za-z])");
```

Comment:
Please move all of this to its own method in Utils.java, it fits better there. This class is for the potion effects feature only.

Reply:
Done! Applied the change in https://github.com/BiscuitDevelopment/SkyblockAddons/pull/361/commits/ff75803b8217a549df402477bff95f6acc50fa87

It parses the tab list every second. `PlayerListener#onTick` invokes the `Utils#parseTablist` every second.
And, the `TabEffectManager` doesn't have any change in this PR.